frontend/src/storage/store.ts

```diff
@@ -0,0 +1,59 @@
+import { makeVar } from "@apollo/client";
+
+export const facilities = makeVar<FacilitiesState>({ current: null, list: [] });
+
+export const appConfig = makeVar<AppConfigState>({
+  dataLoaded: false,
+  user: {
+    id: "",
+    firstName: "",
+    middleName: "",
+    lastName: "",
+    suffix: "",
+    email: "",
+    permissions: [],
+    roleDescription: "",
+    isAdmin: false,
+  },
+  organization: {
+    name: "",
+  },
+  activationToken: null,
+});
+
+export const patient = makeVar<Patient>({
```

Comment:
I am a little worried about the data implications of defaulting to a mostly empty patient. It seems that the patient in the store should be set to `null` when there is no patient

Reply:
Good point.